docs/wustl_htcf.md

```diff
@@ -0,0 +1,21 @@
+# nf-core/configs: WUSTL High Throughput Computing Facility Configuration
+
+Forked from https://github.com/nf-core/configs/blob/master/docs/prince.md
+
+## nf-core pipelines that use this repo
+
+All nf-core pipelines that use this config repo (which is most), can be run on the HTCF. **Before** running a pipeline for the first time, go into an interactive slurm session on a compute node (`srun --pty --time=02:00:00 -c 2`), as the docker image for the pipeline will need to be pulled and converted.
+
+Now, run the pipeline of your choice with `-profile wustl_htcf`. This will download and launch wustl_htcf.config which has been pre-configured with a setup suitable for the HTCF cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a singularity image before execution of the pipeline. This step **takes time**!!
+An example commandline:
+
+`nextflow run nf-core/<pipeline name> -profile wustl_htcf <additional flags>`
+
+## nf-core pipelines that do not use this repo
+
+If the pipeline has not yet been configured to use this config, then you will have to do it manually.
+git clone this repo (https://github.com/nf-core/configs), copy the `wustl_htcf.config` from the conf folder and then you can invoke the pipeline like this:
```

Comment:
You don't need to copy this (it is not ideal as then it won't pick up any changes if the central one gets updated)

Instead it's better to add the following to the end of `nextflow.config` (e.g. in the [template](https://github.com/nf-core/tools/blob/e5ce6ce20304835bd40f102f038b7e1aadc888b2/nf_core/pipeline-template/nextflow.config#L73-L78), but this is a little more complicated so I give a simplified verison below)

```
    includeConfig "https://raw.githubusercontent.com/nf-core/configs/master/nfcore_custom.config"
```